cmd/nerdctl/run_windows.go

```diff
@@ -71,9 +80,52 @@ func setPlatformOptions(
 		opts = append(opts, oci.WithMemoryLimit(uint64(mem64)))
 	}
 
+	// TODO implement hyper-v isolation
+	isolation, err := cmd.Flags().GetString("isolation")
+	if err != nil {
+		return nil, err
+	}
+	switch isolation {
+	case "host":
+		hpAnnotations := map[string]string{}
+		hpAnnotations[hostProcessContainer] = "true"
+
+		// If user is set we will attempt to start container with that user (must be present on the host)
+		// Otherwise we will inherit permissions from the user that the containerd process is running as
+		user, err := cmd.Flags().GetString("user")
+		if err != nil {
+			return nil, err
+		}
+		if user == "" {
+			hpAnnotations[hostProcessInheritUser] = "true"
+		}
+
+		opts = append(opts, oci.WithAnnotations(hpAnnotations))
+	case "default":
```

Comment:
nit: should we just move the "default" case right above `default:` and use [fallthrough](https://stackoverflow.com/questions/40821855/do-go-switch-cases-fallthrough-or-not)